Dockerfile.arm

```diff
@@ -0,0 +1,56 @@
+# Dockerfile for creating an image for testing python versions
```

Comment:
Can we put this in the docs (https://featuretools.alteryx.com/en/stable/install.html) or somewhere else? If possible, I would like to keep the root directory clean and minimal.

Reply:
How about a `docker` or `dockerfiles` subdirectory?  Or, I can remove it and update gitignore to not include it

Reply:
I removed it and will keep it locally for my own use

Reply:
I removed it and will keep it locally for my own use

Reply:
That works.

Another option is to put it in [contributing.md](https://github.com/alteryx/featuretools/blob/main/contributing.md).